ui/dmt_yieldcalculator.lua

```diff
@@ -494,7 +522,7 @@ end
 function IsAdjacentToLandPlot(playerID, x, y)
     -- Cannot use plot:IsAdjacentToLand() since it'll include unrevealed plots.
     local adjPlots = Map.GetAdjacentPlots(x, y);
-    for i, plot in ipairs(adjPlots) do
+    for i, plot in pairs(adjPlots) do
```

Comment:
I looked back through where I changed `ipairs` to `pairs` and it looks like it's all good in terms of NPEs. All of them check if the `plot` value is nil or not, so that part should be covered.

If I understand the existing code correctly, it doesn't seem like the order matters at all in any of these cases. However, if I'm wrong and it does matter somewhere, we could certainly do an alternative. We could try the one you proposed, or maybe do our own `GetAdjacentPlots` function. From my experience, it seems the `Map.GetAdjacentPlots` function returns them in order from the plot to the top left of the current plot, and then goes counter clockwise. Replicating this behavior should be trivial.

If any case, I haven't noticed any issues in regards to these particular changes in my playtests, but I'd be happy to make some adjustments if you'd like!